integreat_cms/cms/templates/analytics/__page_hix_row.html

```diff
@@ -0,0 +1,22 @@
+{% load i18n %}
+{% load rules %}
+{% load content_filters %}
+{% load page_filters %}
+{% with page_translation.page.id as page_id %}
+    {% with page_translation.hix_score as hix %}
+        <tr class="{% if hix < hix_threshold %}bg-red-300{% endif %}">
+            <td>
+                <a href="{% url 'edit_page' region_slug=request.region.slug language_slug="de" page_id=page_id %}">
+                    {{ page_translation.title }}
+                </a>
+            </td>
+            <td>
+                {% if hix %}
+                    {{ hix }}
+                {% else %}
+                    {% translate "This page has no HIX value" %}
```

Comment:
I think we have to think about a bit more on how we want to handle empty pages in this list. I think there are probably two options: a. excluding them from this list, because empty pages are IMO are different problem than too low hix score, or b. leave it as it is. If choose to go with b. I think we should also think about whether or not we want to highlight these pages, and if so how we want to highlight them. These are probably questions for @osmers and @hauf-toni

Reply:
I would suggest to just not show pages with a hix value of `None` and filter them out already in the dashboard context function.